packages/orchestrator-ui-components/src/components/WfoTable/WfoTable/WfoDragHandler.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
```

Comment:
Before merging: verify if the sort buttons are still working, it didn't work on the preview link (but that could be a data issue)